### PR TITLE
add missing optional flag in some qgs expression helps

### PR DESCRIPTION
**Pull request by Gustry**
add missing optional flag in some qgs expression helps

### CODE REVIEW

**Review by nyalldawson — APPROVED** [review]

**Review by nyalldawson — APPROVED** [review]

**Review by nyalldawson — APPROVED** [review]

File: resources/function_help/json/wordwrap [hunk 1]
```diff
@@ -2,10 +2,13 @@
...
-  "arguments": [ {"arg":"string","description":"the string to be wrapped"},
-                 {"arg":"wrap_length","description":"an integer. If wrap_length is positive the number represents the ideal maximum number of characters to wrap; if negative, the number represents the minimum number of characters to wrap."},
-                 {"arg":"delimiter_string","description":"the delimiter string to wrap to a new line (optional)."}
-                 ],
-  "examples": [ { "expression":"wordwrap('UNIVERSITY OF QGIS',13)", "returns":"'UNIVERSITY OF<br>QGIS'"},
-  { "expression":"wordwrap('UNIVERSITY OF QGIS',-3)", "returns":"'UNIVERSITY<br>OF QGIS'"} ]
+  "arguments": [
+    {"arg":"string","description":"the string to be wrapped"},
+    {"arg":"wrap_length","description":"an integer. If wrap_length is positive the number represents the ideal maximum number of characters to wrap; if negative, the number represents the minimum number of characters to wrap."},
+    {"arg":"delimiter_string","optional": true, "description":"the delimiter string to wrap to a new line (optional)."}
```
Comment (DelazJ) [inline resources/function_help/json/wordwrap]: is the (optional) at the end required? or, to mimic the other descriptions
```suggestion
    {"arg":"delimiter_string","optional": true, "description":"Optional delimiter string to wrap to a new line."}
```

**Review by DelazJ — COMMENTED** [review]
Suggesting to add default value to arguments (see eg [generate-series](https://github.com/qgis/QGIS/blob/05535ba9ecee49d9836b3a632db2f2a9599de60a/resources/function_help/json/generate_series)) and playing with Github new suggestion feature...

**Comment by Gustry** [comment]
Thanks @DelazJ. It's just missing a squash all suggestions in one commit...
I guess one of the developer will press the squash button.
If needed I can do it on local.

**Comment by DelazJ** [comment]
Still an experimental feature I guess...

**Pull request merged**